tests/resources/execution_module/action_wrapper.js

```diff
@@ -1,5 +1,32 @@
 const { By } = require('selenium-webdriver')
+var hash = require('object-hash')
+var iterationCollection = new Map()
 
+function GetFeeder(value){
+    if (!iterationCollection.has(hash(value))){
+        iterationCollection.set(hash(value),Array.from(value))
+    }
+    if (iterationCollection.has(hash(value))){
+        var checkCollection = iterationCollection.get(hash(value))
+        if (checkCollection.length <= 0){
+            iterationCollection.set(hash(value),Array.from(value))
+        }
+    }
+    return iterationCollection.get(hash(value)).shift()
+}
+function GetFeederInsideString(value){
```

Comment:
The function is pretty the same as GetFeeder. Please try to replace

Reply:
function was fixed